doc/rst/conf.py

```diff
@@ -54,15 +54,15 @@
 # built documents.
 #
 # The short X.Y version.
-# version = '1.15'
+# version = '1.16'
 
 # We use a custom version variable (shortversion) instead, because setting
 # 'version' adds a redundant version number onto the top of the sidebar
 # automatically (rtd-theme). We also don't use |version| anywhere in rst
-shortversion = 1.15
+shortversion = "1.16 pre-release".replace('-', '&#8209')
```

Comment:
Also, did you decide against having `conf.py` parse `chpl --version`, or were you planning to do that in a future commit?